ietf-dhcpv6-server@2017-11-24.yang

```diff
@@ -1,6 +1,7 @@
-module ietf-dhcpv6-server {
-  namespace "urn:ietf:params:xml:ns:yang:ietf-dhcpv6-server";
-  prefix "dhcpv6-server";
+module ietf-dhcpv6-client {
```

Comment:
client module?

Reply:
Sorry about my careless when copying and pasting the code. I have just started a new pull request.